doc/dirvish.txt

```diff
@@ -167,9 +167,9 @@ Use |:global| to delete the paths with dot-prefixed files: >
     :g@\v/\.[^\/]+/?$@d
 To "toggle", just reload.
 To make this automatic, set |g:dirvish_mode|: >
-    let g:dirvish_mode = ':silent keeppatterns g@\v/\.[^\/]+/?$@d'
```

Comment:
`@d _` should be equivalent. Could you try it?

Reply:
Indeed, I missed that in the docs...

Thanks!